### PR TITLE
[Refactoring] Create new enum for bids and asks and delete useless code

### PR DESCRIPTION
**Pull request by Tim2313**

### CODE REVIEW

File: src/main/java/com/ua/test_task/model/BidOrAskOperation.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package com.ua.test_task.model;
+
+public enum BidOrAskOperation {
```
Comment (MLGMAG) [inline src/main/java/com/ua/test_task/model/BidOrAskOperation.java:3]: Rename to `UpdateOperationType`

File: src/main/java/com/ua/test_task/service/ReaderService.java [hunk 2]
```diff
@@ -13,15 +13,15 @@ public ReaderService(String fileToRead) {
         File file = filepath.toFile();
 
         FileReader fileReader;
-        BufferedReader bufferedReader;
+        BufferedReader buffReader;
```
Comment (MLGMAG) [inline src/main/java/com/ua/test_task/service/ReaderService.java:16]: Change it back to `bufferedReader`

File: src/main/java/com/ua/test_task/service/operation/QueryOperationService.java
Comment (MLGMAG) [inline src/main/java/com/ua/test_task/service/operation/QueryOperationService.java:1]: There is a typo on lie 25 (`splitline`)

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: src/main/java/com/ua/test_task/model/BidOrAskOperation.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+    BID("bid"), ASK("ask");
+
+    private final String type;
```
Comment (MLGMAG) [inline src/main/java/com/ua/test_task/model/BidOrAskOperation.java:7]: Change it to `title`

File: src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java [hunk 4]
```diff
@@ -75,4 +76,16 @@ public static OrderOperationType parseOrderOperationType(String type) {
         }
         return null;
     }
+
+    public static BidOrAskOperation parseUpdateOperation(String type) {
```
Comment (MLGMAG) [inline src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java:80]: Change the method title to `parseUpdateOperationType`

File: src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java [hunk 5]
```diff
@@ -75,4 +76,16 @@ public static OrderOperationType parseOrderOperationType(String type) {
...
+        BidOrAskOperation[] allUpdateOperationTypes = BidOrAskOperation.values();
+
+        for (BidOrAskOperation bidOrAskOperation : allUpdateOperationTypes) {
```
Comment (MLGMAG) [inline src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java:83]: Replace for loop with hashMap

File: src/main/java/com/ua/test_task/service/ReaderService.java (hunk 2 above)
Comment (Tim2313) [reply src/main/java/com/ua/test_task/service/ReaderService.java:16]: fix

File: src/main/java/com/ua/test_task/model/BidOrAskOperation.java (hunk 1 above)
Comment (Tim2313) [reply src/main/java/com/ua/test_task/model/BidOrAskOperation.java:3]: fix

File: src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java
[the remark this answers is not in the document]
Comment (Tim2313) [reply src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java:1]: fix

File: src/main/java/com/ua/test_task/service/operation/QueryOperationService.java
Comment (Tim2313) [reply src/main/java/com/ua/test_task/service/operation/QueryOperationService.java:1]: fix

File: src/main/java/com/ua/test_task/model/BidOrAskOperation.java (hunk 3 above)
Comment (Tim2313) [reply src/main/java/com/ua/test_task/model/BidOrAskOperation.java:7]: fix

File: src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java (hunk 4 above)
Comment (Tim2313) [reply src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java:80]: fix

File: src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java (hunk 5 above)
Comment (Tim2313) [reply src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java]: fix

File: src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java [hunk 6]
```diff
@@ -2,16 +2,14 @@
...
-import java.util.List;
-
 public class OperationInitializationService {
     public static UpdateOperation createUpdateOperation(String line) {
         String[] splitString = line.split(",");
         int price = Integer.parseInt(splitString[1]);
         int size = Integer.parseInt(splitString[2]);
         String type = splitString[3];
-        return new UpdateOperation(price, size, type);
+        UpdateOperationType bidOrAskOperation = parseUpdateOperationType(type);
```
Comment (MLGMAG) [inline src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java:11]: Change the variable name

**Review by MLGMAG — CHANGES_REQUESTED** [review]

**Review by MLGMAG — APPROVED** [review]

File: src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java (hunk 6 above)
Comment (Tim2313) [reply src/main/java/com/ua/test_task/service/operation/OperationInitializationService.java]: fix

**Pull request merged**